src/utils/WellKnownUtils.ts

```diff
@@ -70,6 +75,16 @@ export function tileServerFromWellKnown(
     );
 }
 
+export function getEmbeddedPagesWellKnown(): IEmbeddedPagesWellKnown | undefined {
+    return embeddedPagesFromWellKnown(MatrixClientPeg.get().getClientWellKnown());
```

Comment:
```suggestion
    return embeddedPagesFromWellKnown(MatrixClientPeg.get()?.getClientWellKnown());
```

Reply:
I guess this is needed for future cases which might call this before logging in.

Reply:
The question mark makes an optional parameter